### PR TITLE
Fix tests and add recent Python releases

### PR DESCRIPTION
**Pull request by felixonmars**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py34,py35,py36,pep8,pypy
+envlist = py27,py34,py35,py36,py37,py38,pep8,pypy
```
Comment (jd) [inline tox.ini:2]: you might want to remove py34 and py35 we don't support.

File: tox.ini [hunk 2]
```diff
@@ -10,8 +10,8 @@ deps = .[test,ceph,gnocchi]
        http://tarballs.openstack.org/swift/swift-master.tar.gz#egg=swift
        python-swiftclient
        tooz[redis]
-       http://tarballs.openstack.org/keystone/keystone-master.tar.gz
        httpbin
+       py36,py37,py38: http://tarballs.openstack.org/keystone/keystone-master.tar.gz
```
Comment (jd) [inline tox.ini:14]: you can also use `!py27` which might be handier.

**Comment by jd** [comment]
Thanks @felixonmars !

File: tox.ini (hunk 2 above)
Comment (felixonmars) [reply tox.ini]: Indeed :)

File: tox.ini (hunk 1 above)
Comment (felixonmars) [reply tox.ini]: Sure, updated.

**Comment by felixonmars** [comment]
@jd I'm currently out of ideas of the py27 failure. Do you still want to support Python 2.7 even after its deprecation on Jan 2020?

**Comment by jd** [comment]
@felixonmars no, go ahead and remove Python 2.7 support.

**Comment by felixonmars** [comment]
@jd Tests are passing and Python 2 support dropped along with `six`.

File: setup.cfg [hunk 3]
```diff
@@ -10,8 +10,6 @@ classifier =
     License :: OSI Approved :: Apache Software License
     Operating System :: POSIX
     Programming Language :: Python
-    Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3.6
```
Comment (jd) [inline setup.cfg:14]: Add 3.7 and 3.8?

**Review by jd — CHANGES_REQUESTED** [review]
Perfect, just a little thing. Thanks a ton!

File: setup.cfg (hunk 3 above)
Comment (felixonmars) [reply setup.cfg:14]: Added :)

**Review by jd — APPROVED** [review]

**Pull request merged**